app/controllers/website_controller.rb

```diff
@@ -28,8 +28,49 @@ def terms
     @terms = Consent.terms.active
   end
 
+  def new_forgot_password
+  end
+
+  def create_forgot_password
+    user = User.find_by(email: forgot_password_params[:email])
+
+    if user
+      forgot_password = ForgotPassword.create(user: user)
+      UserMailer.password_recovery(user, forgot_password.token).deliver
+      redirect_to sign_in_path
```

Comment:
Create page with message "An email has been sent with instructions to reset your password"